content/docs/1.13/performance-tuning.md

```diff
@@ -0,0 +1,91 @@
+
+When the instrumented application is generating a large number of spans and the agent/collector is close to the application, the networking overhead might be low, justifying a higher number of flush operations. When the `HttpSender` is being used and the collector is not close enough to the application, the networking overhead might be too high so that a higher value for this property makes sense.
+
+## Adjusting the agent settings
+
+### Server queue sizes
+
+The set of “server queue size” properties ( `processor.jaeger-binary.server-queue-size`, `processor.jaeger-compact.server-queue-size`, `processor.zipkin-compact.server-queue-size`) indicate the maximum number of span batches that the agent can accept and store in memory. It’s safe to assume that `jaeger-compact` is the most important processor in your agent setup, as it’s the only one available in most clients, such as the Java and Go clients.
+
+The default value for each queue is `1000` span batches. Given that each span batch has up to 64KiB worth of spans, each queue can hold up to 64MiB worth of spans.
+
+In typical scenarios, the queue will be close to empty (metric `jaeger_agent_thrift_udp_server_queue_size`) as span batches should be quickly picked up and processed by a worker. However, sudden spikes in the number of span batches submitted by clients might occur, causing the batches to be queued. When the queue is full, the older batches are overridden causing spans to be discarded (metric `jaeger_agent_thrift_udp_server_packets_dropped_total`).
+
+### Processor workers
+
+The set of “processor workers” properties ( `processor.jaeger-binary.workers`, `processor.jaeger-compact.workers`, `processor.zipkin-compact.workers`) indicate the number of parallel span batch processors to start. Each worker type has a default size of `10`. In general, span batches are processed as soon as they are placed in the server queue and will block a worker until the whole packet is sent to the collector. For agents processing data from multiple clients, the number of workers should be increased. Given that the cost of each worker is low, a good rule of thumb is 10 workers per client with moderate traffic: given that each span batch might contain up to 64KiB worth of spans, it means that 10 workers are able to send about 640KiB concurrently to a collector.
+
+## Collector settings
+
+### Queue size
+
+Similar to the Agent, the Collector is able to receive spans and place them in an internal queue for processing. This allows the collector to return immediately to the client/agent instead of waiting for the span to make its way to the storage.
```

Comment:
do we want to always use capitalized works for component names?  We're pretty inconsistent throughout the documentation.

Reply:
Good question. I'd vote for keeping the initials capitalized.